rust/src/pet/participant.rs

```diff
@@ -21,29 +25,55 @@ enum Task {
 /// A participant in the PET protocol layer.
 pub struct Participant {
     // credentials
-    pub encr_pk: box_::PublicKey,      // 32 bytes
+    encr_pk: box_::PublicKey,          // 32 bytes
     encr_sk: box_::SecretKey,          // 32 bytes
-    pub sign_pk: sign::PublicKey,      // 32 bytes
+    sign_pk: sign::PublicKey,          // 32 bytes
     sign_sk: sign::SecretKey,          // 64 bytes
-    pub ephm_pk: box_::PublicKey,      // 32 bytes
+    ephm_pk: box_::PublicKey,          // 32 bytes
     ephm_sk: box_::SecretKey,          // 32 bytes
     certificate: Vec<u8>,              // 0 bytes (dummy)
     signature_sum: sign::Signature,    // 64 bytes
     signature_update: sign::Signature, // 64 bytes
 
-    // other
+    // round parameters
     task: Task,
 }
 
+impl Default for Participant {
+    fn default() -> Self {
```

Comment:
Like for `Coordinator`, I think this should be move to a `new()` method instead of using the `Default` trait.

Reply:
i'll split that.